adafruit_scd30.py

```diff
@@ -314,11 +321,11 @@ def _read_data(self):
             ">f", self._buffer[12:14] + self._buffer[15:17]
         )[0]
 
-    def _check_crc(self, data_bytes, crc):
+    def _check_crc(self, data_bytes: bytearray, crc: int) -> bool:
         return crc == self._crc8(bytearray(data_bytes))
 
     @staticmethod
-    def _crc8(buffer):
+    def _crc8(buffer: bytearray) -> int:
```

Comment:
I'm tempted to say to annotate `buffer` as `bytearray`, but it then might cause problems later since everything if a more specific type was needed later, and now it .  So this is one of the exceptions where I think being more specific is beneficial, so let's keep it as `bytearray`.